packages/dx-react-grid-material-ui/src/plugins/table-row-detail.jsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { TableRowDetail as TableRowDetailBase } from '@devexpress/dx-react-grid';
+import { TableDetailToggleCell } from '../templates/table-detail-toggle-cell';
+import { TableDetailCell } from '../templates/table-detail-cell';
+
+const detailToggleTemplate = props => <TableDetailToggleCell {...props} />;
+const detailCellTemplate = props => <TableDetailCell {...props} />;
+
+export const TableRowDetail = props => (
+  <TableRowDetailBase
+    detailToggleTemplate={detailToggleTemplate}
```

Comment:
Seems that specification of the default width should be like default templates. Don't use defaultProps.